api_app/services/health_checker.py

```diff
@@ -50,7 +50,7 @@ async def create_service_bus_status(credential) -> Tuple[StatusEnum, str]:
     try:
         service_bus_client = ServiceBusClient(config.SERVICE_BUS_FULLY_QUALIFIED_NAMESPACE, credential, retry_total=0)
         async with service_bus_client:
-            receiver = service_bus_client.get_queue_receiver(queue_name=config.SERVICE_BUS_DEPLOYMENT_STATUS_UPDATE_QUEUE)
```

Comment:
Why has this changed?

Reply:
because this was previously connecting to the status update queue in order to test connectivity for the health endpoint - but since that now uses sessions the mechanism to connect to it is more complex - so I opted to move this code to connect to the other 'normal' queue, ie. the airlock one.